sbin/build-redisjson

```diff
@@ -54,6 +54,12 @@ $OP cd RedisJSON
 runn git checkout $BRANCH
 runn git pull --quiet --recurse-submodules
 
+# Patch RedisJSON to build in Alpine - disable static linking
+OS_NAME=$(cat /etc/os-release | egrep "^NAME=" | cut -d '=' -f 2 | tr -d '"')
+if [[ $OS_NAME == "Alpine Linux" ]]; then
+	$OP sed -i "s/^RUST_FLAGS=$/RUST_FLAGS=-C target-feature=-crt-static/g" Makefile
```

Comment:
Please explain this is only fixing building of JSON which is used for test
See [work around](https://github.com/rust-lang/rust/pull/58575#issuecomment-496026747)